### PR TITLE
Dcmstack

### PR DESCRIPTION
**Pull request by TheChymera**
Software package that builds upon nibabel to convert DICOM to NifTi files.

### CODE REVIEW

File: sci-biology/dcmstack/dcmstack-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 )
+DISTUTILS_NO_PARALLEL_BUILD=true
```
Comment (jlec) [inline sci-biology/dcmstack/dcmstack-9999.ebuild]: This is obsolete.

File: sci-biology/dcmstack/dcmstack-9999.ebuild [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+inherit distutils-r1 multilib git-r3 flag-o-matic
```
Comment (jlec) [inline sci-biology/dcmstack/dcmstack-9999.ebuild]: Please sort this alphabetically.

File: sci-biology/dcmstack/dcmstack-9999.ebuild [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+EGIT_REPO_URI="https://github.com/moloney/dcmstack"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE=""
+KEYWORDS=""
+
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]"
+RDEPEND="${DEPEND}
```
Comment (jlec) [inline sci-biology/dcmstack/dcmstack-9999.ebuild]: No need to have setuptools during runtime

File: sci-biology/dcmstack/dcmstack-9999.ebuild [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+	sci-libs/nibabel[${PYTHON_USEDEP}]
+	>=sci-libs/pydicom-0.9.7[${PYTHON_USEDEP}]"
+
+python_prepare_all() {
+	distutils-r1_python_prepare_all
+}
```
Comment (jlec) [inline sci-biology/dcmstack/dcmstack-9999.ebuild]: No need to declare this.

**Comment by TheChymera** [comment]
@jlec is this ready to be pulled now?

**Pull request merged**